Dockerfile

```diff
@@ -24,6 +24,11 @@ ADD https://raw.githubusercontent.com/crops/extsdk-container/master/restrict_use
 COPY poky-entry.py poky-launch.sh /usr/bin/
 COPY sudoers.usersetup /etc/
 
+# For ubuntu, do not use dash.
+RUN [ -f "$(which dash)" ] && (\
```

Comment:
Since "dash" doesn't exist on the non-ubuntu distros, it seems to break all of those distros fail the tests on travis since the RUN command fails. https://travis-ci.org/crops/poky-container/jobs/580365097#L891

Reply:
Good point.  I think this can be done more simply just by `which dash && ...`.  Working on this now.